neofetch

```diff
@@ -78,6 +78,12 @@ get_distro() {
                     *) distro="Red Star OS $(awk -F'[^0-9*]' '$0=$2' /etc/redstar-release)"
                 esac
 
+            elif [[ -f "/etc/siduction-version" ]]; then
```

Comment:
Without this block it does unfortunately not work :confused:.
Here is the contents of  the `/etc/os-release` file from Siduction:

```
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION="9 (stretch)"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```
Please note that the contents of this file is misleading as Siduction is based on the Debian Sid/Unstable branch. That is probably also why Siduction uses the file `/etc/siduction-version` to declare that it's a derivative of Debian Sid. Overall, the Siduction developers told me on IRC that is the proper way to declare a Debian-based Distribution.